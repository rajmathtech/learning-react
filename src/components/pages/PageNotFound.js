import React from 'react';

const PageNotFound = () => (<div> <h2> 404- page not found! </h2> </div>);

export default PageNotFound;