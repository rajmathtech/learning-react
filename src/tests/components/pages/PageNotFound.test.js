import React from 'react';
import {shallow} from 'enzyme';
import PageNotFound from '../../../components/pages/PageNotFound';
test('should render component', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();   
});