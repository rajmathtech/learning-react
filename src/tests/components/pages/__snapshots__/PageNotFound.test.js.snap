// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div>
   
  <h2>
     404- page not found! 
  </h2>
   
</div>
`;
